### PR TITLE
Update packages for react-to-svg-component library

### PR DESCRIPTION
**Pull request by paulofernando**
# [ Task ](https://app.asana.com/0/133999017717994/1203637043480643)

### Acceptance Criteria

1 - Upgrade npm packages

### Tests results

Screen Shot 2023-02-03 at 09 37 44

# TestPlan - [QA steps from notion](https://www.notion.so/publitas-ws/Regression-28ce55f3f16647a89e3bf1ca095b8793)

- [x] Regression on parts of the system that uses react-to-svg-component

# Checklist

- [x] has the change been reviewed by PM / UX
- [x] have infrastructure changes been synced with ops?
- [x] have we added automated tests?
- [x] have we done manual QA (list QA steps)
- [x] have reader changes been tested on all platforms (desktop/mobile/tablet)?
- [x] have we sync translations with wti
- [x] has the code been reviewed and approved
- [x] has the change been tested by the QA team
- [x] does the commit message follow https://chris.beams.io/posts/git-commit/ ?
- [x] does the commit message contain the ACs?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
   },
   "homepage": "https://github.com/publitas/react-to-svg-component#readme",
   "dependencies": {
-    "babel-core": "^6.26.3",
+    "babel-core": "^4.7.16",
```
Comment (felixhageloh) [inline package.json]: just double checking: this looks like it was downgraded instead of upgraded

Comment (paulofernando) [reply package.json]: Oh, good catch, @felixhageloh! I hadn't noticed that.
I wonder why npm is doing that. It is not the first time. I will fix that.

**Comment by paulofernando** [comment]
`Babel` upgraded to version 7:
https://babeljs.io/docs/en/v7-migration
https://github.com/babel/babel-upgrade

Also, `tap` needed to be upgraded.

Screen Shot 2023-02-01 at 08 26 03

**Comment by felixhageloh** [comment]
this repo has not CI integration, so to double check: do tests still pass?

**Comment by paulofernando** [comment]
> this repo has not CI integration, so to double check: do tests still pass?

It is not passing, Felix.
We also needed to update `tap` to a major version, so maybe this new version needs some new kind of configuration. I'll take a look now.

**Review by paolotozzo — APPROVED** [review]

**Review by amansor788 — APPROVED** [review]
looks good, big update tho, lets make sure tests pass for the last version of the branch.
Also (and more important) lets point out to QA spot by spot where this lib is tested in order to cover all regression cases.

**Comment by paulofernando** [comment]
> looks good, big update tho, lets make sure tests pass for the last version of the branch. Also (and more important) lets point out to QA spot by spot where this lib is tested in order to cover all regression cases.

Good point, Ale! I will do that.

**Review by felixhageloh — APPROVED** [review]

**Pull request merged**